.changeset/tame-meals-guess.md

```diff
@@ -0,0 +1,5 @@
+---
+'@keystone-6/fields-document': minor
+---
+
+Added resize to document field editor which allows to scroll through the content when it overflows in `item-view`.
```

Comment:
```suggestion
Changes the editors default overflow behaviour to align with other multi-line text inputs, supporting scrolling instead of an unbounded height for the field.
```